ui/pages/confirmations/hooks/send/useRecipientValidation.test.ts

```diff
@@ -206,7 +206,7 @@ describe('useRecipientValidation', () => {
     jest.spyOn(NameValidation, 'useNameValidation').mockReturnValue({
       validateName: () =>
         Promise.resolve({
-          error: 'ensUnknownError',
```

Comment:
Note to facilitate future stable-sync PR creation: some changes in the [original PR](https://github.com/MetaMask/metamask-extension/pull/36831) are not included here because they were updating "validate bitcoin address for Bitcoin send type" test. This test got introduced in this [PR](https://github.com/MetaMask/metamask-extension/pull/36719), after 13.5.0 release cut, and as a consequence is not included in this release.